include/mbedtls/chacha20.h

```diff
@@ -82,14 +82,15 @@ mbedtls_chacha20_context;
  *                  to \c mbedtls_chacha20_update(), and finally to
  *                  \c mbedtls_chacha20_free().
  *
- * \param ctx       The ChaCha20 context to initialize.
+ * \param ctx       The ChaCha20 context to initialize. Must not be \c NULL.
```

Comment:
As in the other PRs: let's use full sentences, including pronouns.

Reply:
Ok.